src/shared/md/2010-02-12-foonanny.md

```diff
@@ -1,7 +1,13 @@
-Evaluted as “foo” + (+ “bar”), which converts “bar” to not a number.
+Evaluted as `"foo" + (+ "bar")`, which converts “bar” to not a number.
 
 Proposal to rename this site: NaNwtf
 
 <code>
     ("foo" + + "bar") === "fooNaN" // true
-</code>    
\ No newline at end of file
+</code>
+
+**Explanation**
+1. `+"bar"` calls `"bar".valueOf()` which tries to return a number
+2. Because "bar" is a string, it returns `NaN` instead
+3. `"foo"` is concatinated with `NaN`
+4. Because of the concatination `NaN.toString()` is called which results in `"foo" + "NaN"`
```

Comment:
It‘s concatenation and concatenated (L12) :)

Reply:
Oh, right. Thanks! :)